silkworm/silkrpc/core/override_state.cpp

```diff
@@ -84,26 +84,26 @@ evmc::bytes32 OverrideState::read_storage(const evmc::address& address, uint64_t
     return storage_value;
 }
 
-std::optional<silkworm::BlockHeader> OverrideState::read_header(uint64_t block_number, const evmc::bytes32& block_hash) const noexcept {
+std::optional<silkworm::BlockHeader> OverrideState::read_header(BlockNum block_number, const evmc::bytes32& block_hash) const noexcept {
```

Comment:
The same change here and in the rest of functions in the module should be applied also in `override_state.hpp`.

Reply:
ok